accord-core/src/main/java/accord/local/Commands.java

```diff
@@ -856,8 +832,11 @@ static void updateDependencyAndMaybeExecute(SafeCommandStore safeStore, SafeComm
         }
     }
 
-    static void removeWaitingOnKeyAndMaybeExecute(SafeCommandStore safeStore, SafeCommand safeCommand, Key key, boolean executeAsync)
+    static void removeWaitingOnKeyAndMaybeExecute(SafeCommandStore safeStore, SafeCommand safeCommand, Key key)
     {
+        if (safeStore.time().id().id == 3 && safeCommand.txnId().toString().equals("[3,25616,2(KW),3]") && key.toString().startsWith("0:1#"))
```

Comment:
I assume this is for debugging... you do the same as me it seems ^_^

Reply:
heh :) 

usually I use conditional breakpoints but sometimes they're too slow